HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -209,6 +209,7 @@
         'keep *_hltIter4Merged_*_*',
         'keep *_hltIterL3GlbMuon_*_*',
         'keep *_hltIterL3MuonMerged_*_*',
+        'keep *_hltIterL3MuonAndMuonFromL1Merged_*_*',
```

Comment:
This is an auto-generated file, thus changes made directly will be overwritten.
You need to file a JIRA ticket requeting addition of this product!